Gemfile

```diff
@@ -23,10 +23,10 @@ end
 group(:development, :test) do
   gem 'json'
   gem 'lock_manager', *lock_manager_location_for(ENV['LOCK_MANAGER_LOCATION'] || '>= 0')
-  gem 'packaging', github: 'puppetlabs/packaging', branch: '1.0.x'
+  gem 'packaging', :git => 'https://github.com/puppetlabs/packaging', branch: '1.0.x'
```

Comment:
Oh right, I fixed this while I was in here. I got tired of Bundler shouting at me about the insecure `git://` protocol.